doc/response-tagging.rst

```diff
@@ -75,21 +74,21 @@ Only ``/one`` will stay in the cache.
 
 .. note::
 
-    Don't forget to call :ref:`flush <flush>` on the ``CacheInvalidator`` to commit the invalidations
-    to the caching proxy.
-
-.. _custom_tags_header:
```

Comment:
I changed it to `_varnish_custom_tags_header` instead to refer to the right section.